#!/usr/bin/env python
PACKAGE = "teamplay"

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("captureRadius", double_t, 0, "radius in meters", 0.0, 0.0, 10.0)
gen.add("minimumSpeed", double_t, 0, "speed in m/s", 0.0, 0.0, 10.0)

exit(gen.generate(PACKAGE, "teamplayNode", "teamplayInterceptBall"))
